src/shared/directives/conditional-show/domain/conditional-show.model.ts

```diff
@@ -2,42 +2,70 @@ import { _ as _score } from 'underscore';
 
```

Comment:
though it's not in this change but

import { _ as _score } from 'underscore';

is unused import

Reply:
removed